app/code/Magento/Catalog/Helper/Image.php

```diff
@@ -787,7 +776,7 @@ protected function getImageFile()
      */
     protected function parseSize($string)
     {
-        $size = explode('x', strtolower($string));
+        $size = $string ? explode('x', strtolower($string)) : [''];
         if (count($size) == 2) {
```

Comment:
Hello @karyna-tsymbal-atwix 
The `count` function retrieves value of integer type. I think we need using identical comparison here `===`
```suggestion
        if (count($size) === 2) {
```

Reply:
agree, done